docs/tutorials/cloud_fit_deploy/cloud-aws-sagemaker-deployment.md

```diff
@@ -148,17 +162,16 @@ model_data = sagemaker_session.upload_data(
 Prepare transform job:
 
 ```{.python}
-instance_type = "ml.m5.2xlarge"
+instance_type = "ml.m5.2xlarge"  # You might want to use GPU instances for Text/Image/MultiModal Predictors etc
 
 model = AutoGluonInferenceModel(
     model_data=model_data,
     role=role,
     region=region,
-    framework_version="0.4",
+    framework_version="0.5.2",  # Replace this with the AutoGLuon DLC container version you want to use
     py_version="py38",
     instance_type=instance_type,
-    entry_point="tabular_serve-batch.py",
-    source_dir="scripts",
+    entry_point="YOUR_BATCH_SERVE_SCRIPT",
     predictor_cls=AutoGluonTabularPredictor,
```

Comment:
Could we add `# or AutoGluonMultiModalPredictor if model is trained by MultiModalPredictor`?